pkgs/development/libraries/libprom/default.nix

```diff
@@ -0,0 +1,39 @@
+{ gccStdenv
+, fetchFromGitHub
+, cmake
+}:
+let
+  stdenv = gccStdenv;
+in
+stdenv.mkDerivation rec {
+  pname = "libprom";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "digitalocean";
+    repo = "prometheus-client-c";
+    sha256 = "0g69s24xwrv5974acshrhnp6i8rpby8c6bhz15m3d8kpgjw3cm8f";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  doCheck = false;
+
+  patches = [
+    ./prometheus-client-c-0.1.1.patch
```

Comment:
Could you please:

1) Add a comment why this patch is necessary (although I can vaguely guess)

2) Rename it to be more descriptive (e.g. "./fix-install-dir" or something) -- `prometheus-client-c-0.1.1.patch` tells me nothing about its contents, and it's nice for people to be able to skim a derivation and get a rough handle on what it's doing.

Reply:
Done.